src/libs/ReportUtils.js

```diff
@@ -1808,7 +1809,9 @@ function getReportPreviewMessage(report, reportAction = {}, shouldConsiderReceip
     }
 
     const totalAmount = getMoneyRequestReimbursableTotal(report);
-    const payerName = isExpenseReport(report) ? getPolicyName(report) : getDisplayNameForParticipant(report.managerID, true);
+    let displayName = isExpenseReport(report) && isPreviewMessageForParentChatReport ? getPolicyName(report) : getDisplayNameForParticipant(report.managerID, !isPreviewMessageForParentChatReport);
+    displayName = isPreviewMessageForParentChatReport ? displayName : `${displayName}:`;
```

Comment:
WDYT about wrapping this logic in callback inside useMemo? Just to avoid changing value in the variable

Reply:
it's an util function 😃

Reply:
So maybe a function to manage this flow? basically, it's nested ternary operator separated by rewrite a variable. Moe veit to function and if/else instructions clarify what happened in this flow.

Reply:
Yes, it will be moved somewhere else - the plan is to make a bigger refactor. This PR is mainly for getting the values right for the LHN preview. But as I need to work on it a little bit more I guess I can do that here